### ISSUE TITLE
Jakarta in Indonesia

### ISSUE THREAD

**Issue by AngledLuffa** [stanfordnlp/en-foreign-newswire#7]
Phrases like this: one entity or two?

Conll has "Old Trafford in Manchester" as two, but our standard would normally have "Jakarta, Indonesia" as one

**Comment by SecroLoL** [comment]
Thanks for these. I think we need to make a consistent labeling job here. Maybe we can say that if it's just a comma separating them, it should be one entity and if there are any other words then it shouldn't? What are your thoughts

**Comment by SecroLoL** [comment]
Correct. I think the move here is that I'll edit all occurrences that I can find to have entities connected by commas to be one entity span, and then we can have entities that are separated by anything else (e.g. `in`) to be separate spans.
